ee/clickhouse/views/test/test_clickhouse_trends.py

```diff
@@ -928,3 +931,138 @@ def test_aggregating_by_group(self):
         curr_people = get_people_from_url_ok(self.client, data_response["$pageview"]["2020-01-02"].person_url)
 
         assert sorted([p["group_key"] for p in curr_people]) == sorted(["org:5", "org:6"])
+
+
+class ClickhouseTestTrendsCaching(ClickhouseTestMixin, LicensedTestMixin, APIBaseTest):
+    maxDiff = None
+    CLASS_DATA_LEVEL_SETUP = False
+
+    @snapshot_clickhouse_queries
+    def test_insight_trends_merging(self):
+        set_instance_setting("CACHE_HISTORY_TRENDS", "all")
```

Comment:
this should be `strict_caching_enabled` ?

Reply:
nope, `STRICT_CACHING_TEAMS`

Reply:
! why are the snapshots passing if this flag is wrong ...

Reply:
they didn't, they got updated instead 🙈